pubsub/api/Pubsub.Samples.Tests/PubsubFixture.cs

```diff
@@ -256,27 +260,30 @@ public List<Schema> ListSchemaRevisions(string schemaId)
-    }
+    public (string streamArn, string consumerArn, string awsRoleArn, string gcpServiceAccount) KinesisIngestionParams() => ("arn:aws:kinesis:us-west-2:111111111111:stream/fake-stream-name", "arn:aws:kinesis:us-west-2:111111111111:stream/fake-stream-name/consumer/consumer-1:1111111111", AwsRoleArn, GcpServiceAccount);
 
-    public (string clusterArn, string mskTopic, string awsRoleArn, string gcpServiceAccount) RandomAwsMskIngestionParams([CallerMemberName] string caller = null)
-    {
-        var randomName = RandomName();
-        return ($"test{caller}ClusterArn{randomName}", $"test{caller}MskTopic{randomName}", $"test{caller}AwsRoleArn{randomName}", $"test{caller}GcpServiceAccount{randomName}");
-    }
+    public (string clusterArn, string mskTopic, string awsRoleArn, string gcpServiceAccount) AwsMskIngestionParams() => ("arn:aws:kafka:us-east-1:111111111111:cluster/fake-cluster-name/11111111-1111-1", "fake-msk-topic-name", AwsRoleArn, GcpServiceAccount);
+
+    public (string bootstrapServer, string clusterId, string confluentTopic, string identityPoolId, string gcpServiceAccount) ConfluentCloudIngestionParams() => ("fake-bootstrap-server-id.us-south1.gcp.confluent.cloud:9092", "fake-cluster-id", "fake-confluent-topic-name", "fake-identity-pool-id", GcpServiceAccount);
 
-    public (string bootstrapServer, string clusterId, string confluentTopic, string identityPoolId, string gcpServiceAccount) RandomConfluentCloudIngestionParams([CallerMemberName] string caller = null)
+    public (string resourceGroup, string nameSpace, string eventHub, string clientId, string tenantId, string subscriptionId, string gcpServiceAccount) AzureEventHubsIngestionParams() => ("fake-resource-group", "fake-namespace", "fake-event-hub", "11111111-1111-1111-1111-111111111111", "22222222-2222-2222-2222-222222222222", "33333333-3333-3333-3333-333333333333", GcpServiceAccount);
+
+    public async Task InitializeAsync()
     {
-        var randomName = RandomName();
-        return ($"test{caller}BootstrapServer{randomName}", $"test{caller}ClusterId{randomName}", $"test{caller}ConfluentTopic{randomName}", $"test{caller}IdentityPoolId{randomName}", $"test{caller}GcpServiceAccount{randomName}");
+        GoogleCredential appDefaultCredentials = GoogleCredential.GetApplicationDefault();
+        if (appDefaultCredentials.UnderlyingCredential is ServiceAccountCredential sac)
+        {
+            GcpServiceAccount = sac.Id;
+        }
+        else if (appDefaultCredentials.UnderlyingCredential is ComputeCredential cc)
+        {
+            GcpServiceAccount = await cc.GetDefaultServiceAccountEmailAsync();
+        }
+        return;
     }
 
-    public (string resourceGroup, string nameSpace, string eventHub, string clientId, string tenantId, string subscriptionId, string gcpServiceAccount) RandomAzureEventHubsIngestionParams([CallerMemberName] string caller = null)
+    public Task DisposeAsync()
```

Comment:
```suggestion
    public Task DisposeAsync() => Task.CompletedTask;
```
(and delete the rest from below)